src/test/java/com/google/cloud/tools/maven/it/RunAsyncMojoIntegrationTest.java

```diff
@@ -63,6 +67,21 @@ private void test(String name, SupportedDevServerVersion version)
       String urlContent = UrlUtils.getUrlContentWithRetries(getServerUrl(), 60000, 1000);
       assertThat(urlContent, containsString("Hello from the App Engine Standard project."));
       assertThat(urlContent, containsString("TEST_VAR=testVariableValue"));
+
+      Path expectedLog =
+          Paths.get(
+              "target",
+              "test-classes",
+              "projects",
+              "standard-project",
+              "target",
+              "dev-appserver-out",
+              "dev_appserver.out");
+      assertTrue(Files.exists(expectedLog));
+      String devAppServerOutput =
+          new String(Files.readAllBytes(expectedLog), StandardCharsets.UTF_8.name());
```

Comment:
Oh, here the String constructor can take the Charsets as a parameter, so `.name` doesn't need to be called. Only older APIs like `PrintStream` need the string name of the charset.